festival/frontend/src/app/components/news-item/news-item.component.less

```diff
@@ -0,0 +1,27 @@
+.news-item__action {

```

Comment:
you can use less features . See http://lesscss.org/features/#extend-feature-extend-inside-ruleset